app/lib/syndicationService.js

```diff
@@ -0,0 +1,13 @@
+const apiRequest = require('./apiRequest');
+
+const DEFAULT_URL = 'http://v1.syndication.nhschoices.nhs.uk/organisations/gppractices';
+const API_KEY = process.env.SYNDICATION_API_KEY;
```

Comment:
Maybe some error handling to check the key is set? Or perhaps use a library like https://www.npmjs.com/package/require-environment-variables? I've had good experience with the lib